test/elm/interval/test.coffee

```diff
@@ -2124,3 +2167,159 @@ describe 'DecimalIntervalExpand', ->
     # define BadPerMinute: expand { Interval(2.1, 4.1] } per 0.5 minute
     a = @badPerMinute.exec(@ctx)
     should.not.exist(a)
+
+describe 'SameAs', ->
+  @beforeEach ->
+    setup @, data
+
+  it 'returns null when both intervals values are null', ->
+    # define NullBoth: Interval[null,null] same as Interval[null,null]
+    a = @nullBoth.exec(@ctx)
+    should.not.exist(a)
+
+  it 'returns null when one interval is null', ->
```

Comment:
'one interval is null' -> 'one interval's low and high are null' would be more precise